### PR TITLE
[containerapp] `az containerapp ingress cors enable`: Only update arguments `--allow-headers`, `--allow-credentials`, `--allow-methods`, `--expose-methods`, `--max-age` when the value is not `None`

### PR DESCRIPTION
**Pull request by Greedygre**
**Related command**

`az containerapp ingress cors enable`
**Description**

- Fix --max-age input empty string
- Revert [[containerapp] `az containerapp cors enable`: Set the default value of property `--allow-credentials` to `False` by Greedygre · Pull Request #27826 · Azure/azure-cli (github.com)](https://github.com/Azure/azure-cli/pull/27826) . Only update `--allow-credentials` when the value is not `None`.
For command `az containerapp ingress cors enable` argument `--max-age`, accept Empty string to reset max_age to null.
```
--max-age                       : The maximum age of the allowed origin in seconds. Only postive
                                      integer or empty string are allowed. Empty string resets
                                      max_age to null.
```
Currently, when input --max-age "", will throw error:
```
cli.azure.cli.core.azclierror: Bad Request({"type":"https://tools.ietf.org/html/rfc7231#section-6.5.1","title":"One or more validation errors occurred.","status":400,"traceId":"","errors":{"$":["The JSON value could not be converted to System.Nullable`1[System.UInt32]. Path: $ | LineNumber: 0 | BytePositionInLine: 2."]}})
```

**Testing Guide**

**History Notes**

[containerapp] `az containerapp ingress cors enable`: Only update arguments `--allow-headers`, `--allow-credentials`, `--allow-methods`, `--expose-methods`, `--max-age` when the value is not `None`

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
containerapp

File: src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/test_containerapp_commands.py [hunk 1]
```diff
@@ -618,6 +618,20 @@ def test_containerapp_cors_policy(self, resource_group):
             JMESPathCheck('maxAge', 0),
         ])
 
+        self.cmd(
+            'containerapp ingress cors enable -g {} -n {} --allowed-origins "*"  --allow-credentials True --max-age "" '.format(
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/test_containerapp_commands.py:622]: should you add test for the scenario reported

Comment (Greedygre) [reply src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/test_containerapp_commands.py:622]: Already added test for `az containerapp ingress cors ` without --allow-credencial in line 568

Comment (Greedygre) [reply src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/test_containerapp_commands.py:622]: image

**Review by Juliehzl — APPROVED** [review]

**Comment by yonzhan** [comment]
Please fix CI issues.

**Comment by Greedygre** [comment]
Hi @zhoxing-ms 
Can you help to review and merge this PR? Thanks!

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**